tests/LayoutTest.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+namespace EquineSolutions\Filemaker\Tests;
+
+use airmoi\FileMaker\Command\Find;
+use airmoi\FileMaker\FileMaker;
+use EquineSolutions\Filemaker\Exceptions\MethodNotAllowed;
+use EquineSolutions\Filemaker\Layouts\TestLayout;
+use Orchestra\Testbench\TestCase as Orchestra;
+
+class LayoutTest extends Orchestra
+{
+
+    public function tearDown() :void
+    {
+        \Mockery::close();
+    }
+
+    /** @test */
+    public function it_fetches_a_list_of_layout_data()
+    {
+        $filemaker = \Mockery::mock(FileMaker::class);
```

Comment:
Okay nice now I see how this works, I think we should use dependency injection and always the filemaker class as a facade or helper method or just pure dependency injection from the service provider. I think I like the idea of a Facade better so it's more usable and testable. Let me how this affects the architecture.